src/modules/accounting/accounting.py

```diff
@@ -28,4 +32,25 @@ def execute_module(self, blockstamp: BlockStamp):
             self.build_report(report_blockstamp)
 
     def build_report(self, blockstamp: BlockStamp):
-        pass
+        member_info = self.get_member_info(blockstamp)
+        frame_config = self.get_frame_config(blockstamp)
+        chain_config = self.get_chain_config(blockstamp)
+
+        all_validators, lido_keys, lido_validators = self.w3.lido_validators.get_lido_validators_with_others(blockstamp)
+        logger.info({"msg": f"Validators - all: {len(all_validators)} lido: {len(lido_validators)}"})
+
+        cdtp = CommonDataToProcess(
+          blockstamp,
+          blockstamp.slot_number * chain_config.seconds_per_slot + chain_config.genesis_time,
+          blockstamp.slot_number // chain_config.slots_per_epoch,
+          all_validators,
+          lido_validators,
+          lido_keys,
+          member_info.last_report_ref_slot,
+          member_info.last_report_ref_slot // chain_config.slots_per_epoch,
+          (blockstamp.slot_number - member_info.last_report_ref_slot) * chain_config.seconds_per_slot,
+        )
+
+        b = BunkerService(self.w3, frame_config, chain_config)
+        is_bunker = b.is_bunker_mode(cdtp)
```

Comment:
Lets provide here
```
blockstamp
member_info
frame_config
chain_config
```
Rest data you can fetch via w3 or calculate inside service

Reply:
Done